lib/service/storage/elasticsearch.js

```diff
@@ -431,13 +435,18 @@ class ElasticSearch extends Service {
 
     debug('Get document: %o', esRequest);
 
-    return this._client.get(esRequest)
-      .then(({ body }) => ({
+    try {
+      const { body } = await this._client.get(esRequest);
+
+      return {
         _id: body._id,
         _source: body._source,
         _version: body._version
-      }))
-      .catch(error => this._esWrapper.reject(error));
+      };
+    }
+    catch (error) {
+      return this._esWrapper.reject(error);
```

Comment:
You should rather use `throw this._esWrapper.formatESError(error);` to stay consistent with the rest of the codebase.

Also since we are using `æsync/await`, it's better to use `try/catch` rather than resolved or rejected promises.